### ISSUE TITLE
Invalid native attribute names

### ISSUE THREAD

**Issue by yo3jones** [Polymer/polymer-modulizer#292]
Getting this browser error at runtime:

```
Uncaught DOMException: Failed to execute 'setAttribute' on 'Element': 'aria-live\' is not a valid attribute name.
```

It looks like the escape of the `$` is not necessary, which leads to a trailing back slash.

Example can be seen in [iron-a11y-announcer](https://github.com/PolymerElements/iron-a11y-announcer/blob/__auto_generated_3.0_preview/iron-a11y-announcer.js#L13)

```html
<div aria-live\$="[[mode]]">[[_text]]</div>
```

will work if the `\` is removed:
```html
<div aria-live$="[[mode]]">[[_text]]</div>
```

**Comment by venicia** [comment]
Is this actively worked on?

**Comment by stramel** [comment]
Probably not yet as it was opened 20 he's ago. I'm sure it will be unless you want to put out a PR to fix it 🙂

**Comment by usergenic** [comment]
Added https://github.com/Polymer/polymer/issues/5060

**Comment by usergenic** [comment]
I can see this using .raw in Polymer's html-tag may be an issue in every other place involving a `$` or escaped backtick characters etc.

**Comment by usergenic** [comment]
Update: This is actively being worked on in the context of Polymer/polymer 2.next and upcoming 3.0 prerelease version as we speak.  Modulizer output is still going to emit the escaping for forseeable future for all dollar signs, backticks and backslashes.
